### ISSUE TITLE
Icons don't warn about Win10 not supported

### ISSUE THREAD

**Issue by dotMorten** [microsoft/WinUI-Gallery#1557]
### Which version of the app?

WinUI 3 Gallery

### Description

On the Iconography page it's too easy to go pick an icon glyph and add it to your app, not realizing it won't work on Windows 10. For instance the following will render a square placeholder on Windows 10:
```xml
<FontIcon Glyph="&#xEBD3;" />
```

However this will render fine:
```xml
<FontIcon Glyph="&#xE896;" />
```
With so many users still on Win10, it needs to be made easier to avoid making such mistakes, since it can get caught too late in the dev process if the developers are using Win11 machines.

### Screenshots

No indication that this isn't supported on Windows 10:
![image](https://github.com/microsoft/WinUI-Gallery/assets/1378165/b3e1647f-bdc4-411a-9ac5-fd5195fbaac1)
This one right next to it though is supported on Windows 10:
![image](https://github.com/microsoft/WinUI-Gallery/assets/1378165/a87f21e2-2f35-48c8-86b1-980e252e31e3)

### Windows version

Windows 10 22H2 (19045)

**Comment by marcelwgn** [comment]
Thats a good point, would you expect there just to be a "Windows 10" disclaimer or also depending on the Windows build when it was available?

**Comment by niels9001** [comment]
<img width="752" alt="image" src="https://github.com/microsoft/WinUI-Gallery/assets/9866362/8af3e5a1-1202-479d-83b2-978b48401fc4">

It's called out in the Expander. 

But I agree, it doesn't jump out.

Maybe we could add an `InfoBar` at the top calling this out and warning folks that some icons might not render at all and testing is required?

**Comment by dotMorten** [comment]
That message isn’t helpful. Many of the icons do work on Win10 but not all

**Comment by Jay-o-Way** [comment]
Interesting. I wasn't aware some icons wouldn't be available in the MDL2 font. I wonder how many of them are missing. In my opinion, I think the code area (for the specific icon)would be the most appropriate place.

![Screenshot_20240607-001253~2.png](https://github.com/microsoft/WinUI-Gallery/assets/65828559/232b6ff0-684e-486f-80db-e5946cdf9e3d)

**Comment by dotMorten** [comment]
If they are on this page they are supported: https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-ui-symbol-font
If they are on this page, Windows 11 is required: https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font

**Comment by Jay-o-Way** [comment]
> If they are on this page they are supported: https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-ui-symbol-font
> If they are on this page, Windows 11 is required: https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font

Technically, it's not about the OS, but a matter if the Fluent font is installed. Important difference.

**Comment by dotMorten** [comment]
As you said: Technically. In practice though it'll catch developers out.

**Comment by Jay-o-Way** [comment]
Yes, there is a clear correlation between the two. But when the Fluent font is installed on Windows 10 (which is super easy) a message calling out the OS would be invalid. Just trying to keep it accurate.

**Comment by dotMorten** [comment]
> But when the Fluent font is installed on Windows 10 (which is super easy)

Define "super easy". You shouldn't be expecting your users to be doing this. If anything you should include it in your app, but it's still worth pointing out that there's something you need to do and be aware of if you want to use those symbols on Windows 10.

**Comment by Jay-o-Way** [comment]
If I may suggest:

> [!IMPORTANT]
> This icon is not available in the MDL2 font, the default icon font on Windows 10, and thus may not be rendered.

**Comment by Jay-o-Way** [comment]
> Define \"super easy\"

It's literally as simple as installing the font. The OS will pick it up automatically 🙂 But indeed, this is not so common.

**Comment by dotMorten** [comment]
No you don’t just do that. You need to build an app installer that does this. That’s not “super easy”. Again apps shouldn’t require people to do this themselves just to use the app. And that still doesn’t change the fact that someone needs to know they need to ensure this font is available. If it’s not installed all you get is a place holder icon and a poor app experience

**Comment by Jay-o-Way** [comment]
Wait. My comments were based on old ideas. Back when Project Reunion wasn't renamed yet, there was a font (file) that basically acted as MDL2v2. I once installed it and, because of the same internal name, Windows 10 picked it up as the System Icon font. But that's long ago and will not work nowadays. Apologies. 🙇🏻‍♂️

**Comment by HO-COOH** [comment]
Like, what even is this?
![Image](https://github.com/user-attachments/assets/8e9b00e8-8d21-44ca-a682-293d6cdc5761)

**Comment by HO-COOH** [comment]
> [@HO-COOH](https://github.com/HO-COOH) to answer your question: Windows 10 default icon font is MDL2. This font has some less icons than the newer Fluent Icons font, which is default for Windows 11. So, as I said earlier, the app says these icons are available in the Fluent font, so technically there's no blame. It would be a **courtesy** to add a note, for developers building apps that can be run on Windows 10.

Definitely, this should have a warning.
